common/lib/xmodule/xmodule/modulestore/xml.py

```diff
@@ -462,10 +478,10 @@ def load_course(self, course_dir, course_ids, tracker):
                     tracker("'name' is deprecated for module xml.  Please use "
                             "display_name and url_name.")
                 else:
-                    raise ValueError("Can't load a course without a 'url_name' "
-                                     "(or 'name') set.  Set url_name.")
+                    url_name = None
```

Comment:
This raises an error later if the user is importing a course. It's just to allow for hooks in the library case.